src/libxrpl/json/json_reader.cpp

```diff
@@ -953,7 +953,7 @@ operator>>(std::istream& sin, Value& root)
     Json::Reader reader;
     bool ok = reader.parse(sin, root);
 
-    // JSON_ASSERT( ok );
+    // ASSERT( ok );
```

Comment:
```suggestion
```

Commented-out code can be removed. Moreover, if you want to keep it then the `ASSERT` macro must be fully implemented, including giving it a name.